tests/shared/core/training_data/story_reader/test_yaml_story_reader.py

```diff
@@ -611,3 +612,23 @@ def test_read_from_file_skip_validation(monkeypatch: MonkeyPatch):
         _ = reader.read_from_file(yaml_file, skip_validation=False)
 
     assert reader.read_from_file(yaml_file, skip_validation=True) == []
+
+
+def test_raises_exception_missing_intent_in_rules(domain: Domain):
+    rules = "data/test_yaml_stories/rules_missing_intent.yml"
```

Comment:
optional: I'd slightly prefer to inline the stories because it makes the tests easier to read (I don't have to jump to the file to understand the test

Reply:
I'll remember this for future tests 👍